### PR TITLE
Disable RBAC risk due to performance instability

### PR DESCRIPTION
**Pull request by connorgorman**
## Description

12k roles, rolebindings, service accounts
11k deployments

Running for 13 minutes
```
goroutine 1 [chan receive, 13 minutes]:
```
```
process_cpu_throttled_time                                                       3448381546095
```
which is 57 minutes of throttling which explains some of the slowness.

Screen Shot 2022-05-06 at 10 38 30 AM
Screen Shot 2022-05-06 at 10 38 07 AM
Screen Shot 2022-05-06 at 10 38 19 AM

## Checklist
- [ ] Investigated and inspected CI test results
- [ ] Unit test and regression tests added
- [ ] Evaluated and added CHANGELOG entry if required
- [ ] Determined and documented upgrade steps
- [ ] Documented user facing changes (create PR based on [stackrox/openshift-docs](https://github.com/stackrox/openshift-docs) and merge into [rhacs-docs](https://github.com/openshift/openshift-docs/tree/rhacs-docs))

If any of these don't apply, please comment below.

## Testing Performed

CI should pass

### CODE REVIEW

**Review by clickboo — COMMENTED** [review]
This is a customer facing change - in the sense risk scoring and risk priority will change for a customer if they upgrade to a release with this PR right? We should add a changelog and some doc?

I believe this is a purely perf issue with our risk calc algorithm and how we process risk?

**Comment by connorgorman** [comment]
> This is a customer facing change - in the sense risk scoring and risk priority will change for a customer if they upgrade to a release with this PR right? We should add a changelog and some doc?

Yeah I will add a changelog if we think we want to make this change

**Comment by viswajithiii** [comment]
@connorgorman Are you planning to merge this one or just testing?

**Comment by md2119** [comment]
Can we release a patch only for the badly affected customers instead of dropping it entirely?

**Comment by keyallis** [comment]
From my prospective this seems like a change we should be making, and one of the bigger reasons it's been hard to justify is that we didn't have a concrete example of a customer being impacted negatively enough by rbac in risk calculations to the point where it should be removed.
I personally can't see any reasons why removing this would significantly impact customers and lead to the ask of having it added back, but if it seems appropriate to add a feature flag to toggle it's inclusion I would say the default state should be disabled.

**Review by md2119 — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -41,7 +41,8 @@ Please avoid adding duplicate information across this changelog and JIRA/doc inp
   - For questions about this change, please contact the Red Hat support team at support@redhat.com.
 - ROX-10018: The policy `OpenShift: Kubeadmin Secret Accessed` will no longer trigger if the request was from the default OpenShift `oauth-apiserver-sa` service account, because this is an expected access pattern for the OpenShift apiserver.
 - Violation tags and process tags are deprecated, and will be removed in version 3.72.0.
-
+- RBAC calculation is no longer included in Risk by default, since it was not performant. Users who want it included can set
+  the "INCLUDE_RBAC_IN_RISK" to "true" in the Central deployment spec.
```
Comment (clickboo) [inline CHANGELOG.md:45]: I think the default should be true, and set to false if not required? I believe Kirsten had the same feedback. In her exact words:
"Also, it should not be disabled by default so as not to take aways current functionality"

File: CHANGELOG.md [hunk 2]
```diff
@@ -41,7 +41,8 @@ Please avoid adding duplicate information across this changelog and JIRA/doc inp
   - For questions about this change, please contact the Red Hat support team at support@redhat.com.
 - ROX-10018: The policy `OpenShift: Kubeadmin Secret Accessed` will no longer trigger if the request was from the default OpenShift `oauth-apiserver-sa` service account, because this is an expected access pattern for the OpenShift apiserver.
 - Violation tags and process tags are deprecated, and will be removed in version 3.72.0.
-
+- Users who do not want to include the RBAC factor in risk calculation can set
```
Comment (theencee) [inline CHANGELOG.md:44]: Do you want it to be documented at all?

**Review by theencee — APPROVED** [review]

**Review by clickboo — APPROVED** [review]

File: CHANGELOG.md (hunk 2 above)
Comment (viswajithiii) [reply CHANGELOG.md:44]: Yeah, I think it's fine to.

**Pull request merged**